### PR TITLE
A bunch of new words for CyberCode.

### PR DESCRIPTION
**Pull request by codeshark20**
**Edits are on lines:**
5, 11, 37, 56, 60, 99, 126, 215, 275, 291, 336, 386, 398, 453, 493, 521, 526, 537.

_This is my first time contributing, so feel free to revise any changes made by me._

### CODE REVIEW

File: contribution/words.json [hunk 1]
```diff
@@ -2,11 +2,13 @@
 	"miscTasks": [
 		"acting like a bopper",
 		"adding infinity to infinity",
+		"assimilating SL-Database into breaching alogrithms",
```
Comment (svenk-k) [inline contribution/words.json:5]: al**go**rithms instead of al**og**rithms

File: contribution/words.json [hunk 2]
```diff
@@ -383,6 +395,7 @@
 		"Unhoused Person",
 		"Vigilante",
 		"VTuber",
+		"Vitrual Reality Goggles",
```
Comment (svenk-k) [inline contribution/words.json:398]: Vi**rt**ual instead of Vi**tr**ual

Comment (codeshark20) [reply contribution/words.json:398]: Thank you man! I feel like contributing more to this wonderful game.

**Comment by DexterHuang** [comment]
seems like there is formatting error :)

File: contribution/words.json [hunk 3]
```diff
@@ -503,10 +518,12 @@
 		"Want some?",
 		"You got the hands, I got the job to fit them",
 		"you need some BTC?",
+		"You look like you could use a job"
```
Comment (svenk-k) [inline contribution/words.json:521]: If you decide to reopen this or another PR. Here's the formatting error. A comma is missing at the end.

**Pull request closed without merging**